scripts/Editor/DllManipulatorEditor.cs

```diff
@@ -339,5 +362,12 @@ public static void UnloadAll()
         {
             DllManipulator.UnloadAll();
         }
+
+        [NativeDllLoadedTrigger(UseMainThreadQueue = true)]
+        [NativeDllAfterUnloadTrigger(UseMainThreadQueue = true)]
+        public static void RepaintAll()
+        {
+            RepaintAllEditors.Invoke();
```

Comment:
Agreed however as the event is initialized with the default delegate it will never be null. https://stackoverflow.com/q/170907/9295437
Ill add it anyways...